code/modules/reagents/chemistry/recipes.dm

```diff
@@ -60,6 +60,10 @@
 	else
 		new /obj/effect/temp_visual/shockwave(T)
 		playsound(T, 'sound/effects/bang.ogg', 25, 1)
+	// PARADISE EDIT: Allow only a certain amount of atoms to be pulled per unit
+	var/units_per_atom = 5
+	var/atoms_to_move = round(volume / units_per_atom)
+	var/moved_count = 0
```

Comment:
May want to adjust `/datum/reagent/consumable/drink/apple_pocalypse` and `/datum/reagent/consumable/drink/grape_granade` so they actually function, post these changes.